pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -242,12 +247,12 @@ evaluateTerm cont handler env (Conditional c info) = case c of
     -- chargeGasArgs info (GAConstant constantWorkNodeGas)
     evalCEK (CondC env' info (EnforceC str) cont) handler env' cond
   CEnforceOne str conds -> do
-    chargeGasArgs info (GAConstant unconsWorkNodeGas)
     case conds of
       [] ->
         returnCEK cont handler (VError "enforce-one failure" info)
       x:xs -> do
         -- Todo: is this a bit too cheap??
+        chargeGasArgs info (GAConstant unconsWorkNodeGas)
```

Comment:
I moved the gas charging of uncons to the actual uncons, and to make gas reproducible between the two interpreters.

Reply:
Should be fine. The error should consume all gas anyway so we don't care about the particular amount.